lib/sqlalchemy/sql/functions.py

```diff
@@ -1795,3 +1799,21 @@ class grouping_sets(GenericFunction[_T]):
     """
     _has_args = True
     inherit_cache = True
+
+
+class aggregate_strings(GenericFunction[_T]):
+    r"""Implement the ``STRING_AGG`` aggregation function
```

Comment:
**mike bayer** (zzzeek) wrote:

implement a generic string aggregation function.

this function is compiled on a per-backend basis, into functions such as ``group_concat()``, ``string_agg()``, or ``LISTAGG()``.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4781

Reply:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4781